content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImportWaitForVm.class/__methods__/wait_for_imported_vm.rb

```diff
@@ -1,36 +1,42 @@
+#
+# Description: This method checks if the VM has been successfully imported
+#
+
 module ManageIQ
   module Automate
-    module System
-      module Event
-        module EmsEvent
-          module RHEVM
-            class UpdateVmImportStatus
```

Comment:
What happened to updateVmImportStatus? I do not see any information why are you replacing it.